app/templates/components/widget-discussions-post.hbs

```diff
@@ -41,23 +41,23 @@
 	</span>
 	<span class="post-counters">
 		<span class="upvote-count">
-			{{svg 'upvote' viewBox='0 0 24 24' class='icon upvote'}}
+			{{svg 'wds-icons-upvote' viewBox='0 0 24 24' class='wds-icon wds-icon-tiny upvote'}}
 			{{post.upvoteCount}}
 		</span>
 		<span class="reply-count">
-			{{svg 'reply' viewBox='0 0 24 24' class='icon replies'}}
+			{{svg 'wds-icons-reply' viewBox='0 0 24 24' class='wds-icon wds-icon-tiny replies'}}
```

Comment:
why do we use regular size icon and apply wds-icon-tiny class to it? And do we need this viewbox?